examples/README.md

```diff
@@ -1,20 +1,24 @@
 # Examples Gallery
 
-- [Graph types](#graph-types)
-  - [Cloropleth](#cloropleth)
-  - [Diverging Stacked Bar Chart](#diverging-stacked-bar-chart)
-  - [Scatterplot](#scatterplot)
-  - [Stacked Bar Grap](#stacked-bar-graphh)
-  - [Stock Graph](#stock-graph)
-  - [Line with confidence Interval](#line-with-confidence-interval)
-- [Data Sources](#data-sources)
-  - [From ndarray](#from-ndarray)
-  - [From csv](#from-csv)
-  - [From url](#from-url)
-  - [From JSON spec](#from-json-spec)
-  - [From JSON spec with new Data](#from-json-spec-with-new-data)
-- [Miscellaneous](#miscellaneous)
-  - [Without using builders](#without-using-builders)
+- [Examples Gallery](#examples-gallery)
```

Comment:
if you use The same auto-toc plugin than me (part of VSCode Markdown All In One) then you can annotate title to not be part of the toc
```
# Examples Gallery <!-- omit in toc -->
```

Reply:
added this to Thomas Aubry 6420b49